core/src/tab/tab.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import { storiesOf } from "@storybook/react"; //eslint-disable-line
-import React from "react";
+import React, { useState } from "react";
```

Comment:
We already imported "React" so it's better to just use React.useState

Also it should be import * as React